pyi.py

```diff
@@ -46,8 +46,10 @@ def unparse(node: ast.AST) -> str:
 
 if sys.version_info >= (3, 9):
     _LiteralMember: TypeAlias = ast.expr
+    _BuiltinsTypeMember: TypeAlias = ast.expr
 else:
     _LiteralMember: TypeAlias = Union[ast.expr, ast.slice]
+    _BuiltinsTypeMember: TypeAlias = Union[ast.expr, ast.slice]
```

Comment:
Maybe we should define a common type alias, instead of having two type aliases for the same type. `_SliceContents: TypeAlias = ...`?